### PR TITLE
added pods cli redirect

### PR DESCRIPTION
**Pull request by sascala**
## What is the urgency level of this change?
- [ ] Blocker
- [X] High
- [ ] Medium

## I have completed these items:
- [ ] I have built locally and tested for broken links and formatting.
- [X] If appropriate, I have added redirects.

## If you included a comment with your commit, it appears here:

Note: do not merge until this merges: https://github.com/dcos/dcos-docs/pull/782

### CODE REVIEW

File: redirect-files [hunk 1]
```diff
@@ -55,3 +55,10 @@
 /docs/1.8/usage/service-discovery/marathon-lb/quickstart/index.html /docs/1.8/usage/service-discovery/marathon-lb/marathon-lb-basic-tutorial/
 /docs/1.9/usage/service-discovery/marathon-lb/quickstart/index.html /docs/1.9/usage/service-discovery/marathon-lb/marathon-lb-basic-tutorial/
 /docs/1.8/usage/registry/index.html https://github.com/dcos/examples/tree/master/1.8/registry
+/docs/1.9/usage/pods/pods-cli.md /docs/1.9/usage/pods/using-pods/
+n
```
Comment (joel-hamill) [inline redirect-files]: What are lines 59 - 64?

File: redirect-files [hunk 2]
```diff
@@ -56,9 +56,3 @@
 /docs/1.9/usage/service-discovery/marathon-lb/quickstart/index.html /docs/1.9/usage/service-discovery/marathon-lb/marathon-lb-basic-tutorial/
 /docs/1.8/usage/registry/index.html https://github.com/dcos/examples/tree/master/1.8/registry
 /docs/1.9/usage/pods/pods-cli.md /docs/1.9/usage/pods/using-pods/
```
Comment (joel-hamill) [inline redirect-files]: please format the from link like the rest (i.e. `pods-cli/index.html`).

**Comment by joel-hamill** [comment]
ping @sascala

File: redirect-files [hunk 3]
```diff
@@ -57,4 +57,6 @@
 /docs/1.9/administration/user-management/index.html /docs/1.9/administration/id-and-access-mgt/user-management/
 /docs/1.8/usage/service-discovery/marathon-lb/quickstart/index.html /docs/1.8/usage/service-discovery/marathon-lb/marathon-lb-basic-tutorial/
 /docs/1.9/usage/service-discovery/marathon-lb/quickstart/index.html /docs/1.9/usage/service-discovery/marathon-lb/marathon-lb-basic-tutorial/
-/docs/1.8/usage/tutorials/registry/index.html https://github.com/dcos/examples/tree/master/1.8/registry
+/docs/1.8/usage/registry/index.html https://github.com/dcos/examples/tree/master/1.8/registry
```
Comment (joel-hamill) [inline redirect-files]: @sascala where did line 60 come from? was this from a merge conflict resolution? there should be just one entry for registry (per https://github.com/dcos/dcos-website/pull/1006).

**Comment by sascala** [comment]
Oh shoot, OK. Yes, I was resolving a merge conflict. Incorrectly...

**Comment by sascala** [comment]
@joel-hamill Is this correct now?

**Pull request merged**